app/src/main/java/org/hwyl/sexytopo/control/graph/GraphView.java

```diff
@@ -1367,12 +1383,11 @@ private void drawLine(Canvas canvas, Coord2D start, Coord2D end, Paint paint, in
     /**
      * Drawing as a path is useful in order to apply path effects (buggy when drawing lines)
      */
-    private void drawLineAsPath(Canvas canvas, Coord2D start, Coord2D end, Paint paint, int alpha) {
-        paint.setAlpha(alpha);
-        Path path = new Path();
-        path.moveTo((float)(start.x), (float)(start.y));
-        path.lineTo((float)(end.x), (float)(end.y));
-        canvas.drawPath(path, paint);
+    private void drawLineAsPath(Canvas canvas, Coord2D start, Coord2D end, Paint paint) {
```

Comment:
I think we still need the `alpha` or vertical-ish lines in linked surveys will be drawn unfaded?

Reply:
I set that line `905`

Reply:
Not sure I follow, the fade overridden by `baseAlpha` no?

Reply:
I'm probably too tired to be looking at PRs :slightly_smiling_face: 
Yes, think that's correct.